src/main/scala/mesosphere/marathon/core/launcher/OfferProcessorConfig.scala

```diff
@@ -9,6 +9,11 @@ trait OfferProcessorConfig extends ScallopConf {
     descr = "Offer matching timeout (ms). Stop trying to match additional tasks for this offer after this time.",
     default = Some(1000))
 
+  lazy val saveTasksToLaunchTimeout = opt[Int]("save_tasks_to_launch_timeout",
```

Comment:
Since this is rather special, I didn't want to expose this config option at all. That's why it is "hidden".

Maybe it is better to document it. This documentation could be extended as well:

"[...] about to launch. [Added from here] When reaching the timeout, only the tasks that we could already save are also launched. All other task launches are temporarily rejected and retried later."